src/components/modals/AddOrEditMovieForm/Select.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { FormikErrors, FormikTouched } from "formik";
+import styles from "../../../css-modules/modal.module.css";
+
+// Interfaces
+import Movie from "../../../types/movie.interface";
+
+interface Props {
```

Comment:
This interface is almost the same as in `Input` component. It's better to create a one interface for this

Reply:
resolved